src/org-manager/containers/organisation-details/organisation-details.component.ts

```diff
@@ -109,6 +114,11 @@ export class OrganisationDetailsComponent implements OnInit, OnDestroy {
           }
           this.orgs$ = of(organisationVM);
         });
+      if (this.newRegisterOrg) {
+        this.lovRefDataService.getListOfValues(this.CATEGORY_ORGANISATION_TYPE, true).subscribe((orgTypes) => {
```

Comment:
@connorpgpmcelroy - would be good to assign this to some variable that can later be unsubscribed in `ngOnDestroy`, to avoid any resource leaks

Reply:
agree